.github/workflows/master.yml

```diff
@@ -75,6 +75,9 @@ jobs:
     name: Publish Binaries
     runs-on: macos-latest
     needs: build-and-test
+    strategy:
```

Comment:
We were missing the strategy block

Reply:
Do we need it if there's no matrix? i.e. there is only 1 version.

The [documentation](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix) seems to imply it's only necessary if there is more than 1 version/language/whatever that we are testing.

Reply:
I am being cunning here! this means that we have a single place to update the versions of our builds - this will allow me to add multiple go versions / multiple dotnet / python etc soon! So I am kinda future proofing the scripts

Reply:
Makes sense 👍